graylog2-server/src/main/java/org/graylog2/streams/OutputAVImpl.java

```diff
@@ -32,44 +32,52 @@
 @JsonAutoDetect
 @CollectionName("outputs")
 public abstract class OutputAVImpl implements Output {
```

Comment:
So could we rename this class to `OutputImpl` now?

FWIW, I would get rid of the `Output` interface completely since this is a "dumb" DTO now and doesn't contain any logic.